README.md

```diff
@@ -1,59 +1,120 @@
+    <li><a href="https://css.org/">Css</a></li>
+  </ul>
+</details>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+git clone https://github.com/hagan199/my-portfolio.git
+cd Setup-and mobile-first
+npm install
 
-- Create a new repo
-- Create a new branch according to the GitHub flow rules.
-- Add linters
-- Add a git ignore file
-- Add HTML & CSS files
-- Open a pull request
 
 ### Prerequisites
 
-- Git. Which is a open source distributed version control system designed for source code management.
-- A text editor (e.g Visual Studio Code, Vim.)
-- A web browser (e.g Chrome, Safari, Mozilla Firefox.)
-- Live Server which is a web extension that helps you to live reload feature for dynamic content (PHP, Node.js, ASPNET)
+In order to run this project you need:
+- Web browser
+- Have visual studio code installed
+- Install node.js
+- Install git
+- You need to Go Live
+
+### Install Node
+
+Install this project with this command:
+
+  cd my-project
+  npm install
 
-### Setup
+### Usage
 
-- Using git, clone the code to your machine or download a Zip File.
+To run the project, execute the following command:
+
+Open index.html using live server extension
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
 
-👤 **Authors**
 
 👤 Emmanuel Hagan
 
-- GitHub: (https://github.com/hagann109)
-- Twitter: (https://twitter.com/emmahagan199)
-- LinkedIn:(https://linkedin.com/inemma199)
+- GitHub: [hagan199](https://github.com/hagan199)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Develop the Desktop Version
+- Add styling
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
 
-## 🤝 Contributing
+If you like this project please give it a star.
 
-Contributions, issues, and feature requests are welcome!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- Contributions, issues, and feature requests are welcome!
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-- Feel free to check the [https://github.com/hagan199/my-portfolio/issues].
+I would like to thank Microverse and my coding partners with all the resources they provided.
 
-https://github.com/hagan199/my-portfolio/issues
-## Acknowledgments
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- Microverse
-- Inspiration
+<!-- LICENSE -->
 
+## 📝 License <a name="license"></a>
 
-## 📝 License
+This project is [MIT](./MIT.md) licensed.
 
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
 
```

Comment:
[OPTIONAL] Please remove this line because this is only for you not for others so kindly remove this line :slightly_smiling_face: 
```suggestion

```